src/cache-save.ts

```diff
@@ -12,6 +12,13 @@ export async function run(): Promise<void> {
 }
 
 async function cacheRTXTools(): Promise<void> {
+  const skipCache = core.getState('SKIP_CACHE')
```

Comment:
why not use core.getBooleanInput('enable_cache')?

Reply:
I thought I remember reading somewhere that input aren't available in the post script but I can't find that article.  I also can't find any examples that call `getInput` from post scripts so this seemed safer.  I can change it if you have seen it work.

Reply:
oh I see, well in that case let's just name it `CACHE` to match the config setting